packages/esm-patient-flags-app/src/patient-flags/patient-flags.scss

```diff
@@ -0,0 +1,22 @@
+@use '@carbon/styles/scss/spacing';
+@import '~@openmrs/esm-styleguide/src/vars';
+
+.flagContainer {
+    padding: spacing.$spacing-03;
+    border-bottom: 0.063rem solid $ui-03;
```

Comment:
Make it 1px instead of 0.063rem 

In some browsers the border might not even appear if kept in rem